### DISCUSSION TITLE
Check if Mouse in on GLobe in GlobeView

### DISCUSSION THREAD

**Discussion by JanVanDerWall** [visgl/deck.gl#6694] [Q&A]
I am using the GlobeView and the onClick callback if the Deck component. I was wandering if there is an elegant way of checking if the click (or any event) was on the Globe or next to it. I do not have any Layer that is pickable therefore i can not simply check if the layer prop of the pickinfo object is empty.

Thanks in advance.

**Comment by Pessimistress** [comment]
You can render a transparent layer that is pickable.
